src/runtimes/burrego.rs

```diff
@@ -44,16 +44,35 @@ impl<'a> Runtime<'a> {
                 // Policies for OPA expect the whole `AdmissionReview`
                 // object: produce a synthetic external one so
                 // existing OPA policies are compatible.
-                (
-                    json!({
-                        "apiVersion": "admission.k8s.io/v1",
-                        "kind": "AdmissionReview",
-                        "request": &request.0,
-                    }),
-                    json!(settings),
-                )
+                match request {
+                    ValidateRequest::AdmissionRequest(adm_req) => (
+                        json!({
+                            "apiVersion": "admission.k8s.io/v1",
+                            "kind": "AdmissionReview",
```

Comment:
I think you said we can omit these 2 lines. Can you drop them if that's true? That would reduce the amount of code